tests/bson/bson-utcdatetime-serialization_error-004.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+MongoDB\BSON\UTCDateTime unserialization requires "milliseconds" string to parse as 64-bit integer (__serialize and __unserialize)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_php_version('<', '7.4.0'); ?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+echo throws(function() {
+    unserialize('O:24:"MongoDB\BSON\UTCDateTime":1:{s:12:"milliseconds";s:9:"1234.5678";}');
+}, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
+
+/* TODO: Add tests for out-of-range values once CDRIVER-1377 is resolved */
```

Comment:
This TODO was copied from `bson-utcdatetime-serialization_error-002.phpt`. I created [PHPC-1957](https://jira.mongodb.org/browse/PHPC-1957) to track this.